### PR TITLE
fix: qa profile and manager page

### PR DESCRIPTION
**Pull request by ChristanDaniel**
## Why
We identified bugs on the management and profile pages.

## Changes

- add point-event to the active state in Breadcrumb component
- update blog link in Header
- add manager total tvl value in profile page
- add max domain value in chart
- fix pool existence check in the manager page

### CODE REVIEW

File: src/hooks/query/useManagerTotalTVL.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { useQuery } from '@tanstack/react-query'
+import { getAddress } from 'ethers'
+
+import { kassandraClient } from '@/graphQLClients'
+
+type fetchManagerTotalTVL = {
```
Comment (nicholascostadev) [inline src/hooks/query/useManagerTotalTVL.tsx:6]: Capitalize type def

```suggestion
type FetchManagerTotalTVL = {
```

**Review by nicholascostadev — CHANGES_REQUESTED** [review]

File: src/templates/PoolManager/FeeRewards/FeesChart/index.tsx [hunk 2]
```diff
@@ -46,6 +47,14 @@ const monthShort = [
 ]
 
 const FeesChart = ({ fees, title, legend }: Props) => {
+  const maxDomain = React.useMemo(() => {
+    const maxFeesJoinManager = fees.map(
+      item => parseFloat(item.feesJoinManager) * 1.1
```
Comment (An-dz) [inline src/templates/PoolManager/FeeRewards/FeesChart/index.tsx:52]: What is this magic 1.1? Put it in a constant so its name explains what it does.

**Review by An-dz — APPROVED** [review]

File: src/components/Modals/ModalKacy/index.tsx [hunk 3]
```diff
@@ -111,7 +111,7 @@ const ModalKacy = () => {
     }
 
     if (wallet?.provider) {
-      getKacyBalanceInWallet(wallet.accounts[0].address)
+      getKacyBalanceInWallet(wallet?.accounts[0]?.address)
```
Comment (guilherme-correa-s) [inline src/components/Modals/ModalKacy/index.tsx:114]: No need to use optional chaining
```suggestion
      getKacyBalanceInWallet(wallet.accounts[0]?.address)
```

**Review by guilherme-correa-s — APPROVED** [review]
LGTM

**Review by nicholascostadev — APPROVED** [review]

**Review by guilherme-correa-s — APPROVED** [review]

**Pull request merged**